src/monodroid/CMakeLists.txt

```diff
@@ -306,6 +313,15 @@ if(UNIX)
     )
 endif()
 
+add_library(xa-internal-api SHARED ${XA_INTERNAL_API_SOURCES})
+target_link_libraries(xa-internal-api ${EXTRA_LINKER_FLAGS})
+set_target_properties(
+  xa-internal-api
+  PROPERTIES
+  COMPILE_FLAGS -fvisibility=default
+  LINK_FLAGS -fvisibility=default
```

Comment:
Why is this `-fvisibility=default` instead of `-fvisibility=hidden`?

Reply:
this DSO is supposed to have only external symbols, so it's here instead/in addition to `MONO_API`